### ISSUE TITLE
Provide better way to change drag axis

### ISSUE THREAD

**Issue by blackears** [blackears/cyclopsLevelBuilder#48]
Right now XZ is assumed for dragging by default. Provide button to let users choose the axis/plane to drag along instead.  This is better for mobile devices too.

**Comment by elvisish** [comment]
Doesn't one of the hotkeys do this, ala Trenchbroom?

**Comment by Calinou** [comment]
> Doesn't one of the hotkeys do this, ala Trenchbroom?

You can hold <kbd>Alt</kbd>, but it only allows moving on the Y axis: https://github.com/blackears/cyclopsLevelBuilder/issues/48#issuecomment-1546838407

**Comment by elvisish** [comment]
Yeah, I'm having trouble moving a face along the Z axis without it also going off into the X axis, it would be useful if there was a way to just lock to a single axis (I dont think there is?)

**Comment by blackears** [comment]
I've just merged code that provides a gizmo that should make it easier to move you selection along a particular axis.
